### PR TITLE
Gp19 62 create reusable input component

### PR DESCRIPTION
**Pull request by GBurley-Jones**

### CODE REVIEW

File: src/Conference/ConferenceItem.jsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React from "react";
 import styled from "styled-components";
 import { getTimestring, getDatestring, getBackgroundColor } from "../utils";
 import { Link } from "react-router-dom";
+//import Input from "../Input"
```
Comment (dex-mendeley) [inline src/Conference/ConferenceItem.jsx]: If not needed best to remove it ;)

File: src/Conference/ConferenceItem.jsx [hunk 2]
```diff
@@ -48,6 +48,7 @@ const ConferenceItem = ({ conference }) => {
         <p>{city}</p>
         <StyledLink to={`/${id}`}>Description</StyledLink>
       </StyledCard>
+      {/* <Input type="datetime-local" /> */}
```
Comment (dex-mendeley) [inline src/Conference/ConferenceItem.jsx]: Same here

File: src/Input.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+//Use 
+//<Input label="name:" type="text" etc />
```
Comment (dex-mendeley) [inline src/Input.jsx]: And here

File: src/Input.jsx [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+//Use 
+//<Input label="name:" type="text" etc /> 
+const Input = ({ label, type, id, value, onChange, ...props }) => {
+  return (
+    <label>
+      {label}
```
Comment (dex-mendeley) [inline src/Input.jsx]: The usual markup for inputs with labels is this: 

```
<div class="preference">
    <label for="peas">Do you like peas?</label>
    <input type="checkbox" name="peas" id="peas">
</div>
```

File: src/Input.jsx (hunk 3 above)
Comment (GBurley-Jones) [reply src/Input.jsx]: This is just a comment to remind us how to use the input component :) we will delete it once we have properly implemented it!

**Comment by shanassaf** [comment]
Just sort out the conflicts!

**Pull request merged**